lib/src/screens/session/session_screen.dart

```diff
@@ -242,7 +213,7 @@ class _SessionScreenState extends State<SessionScreen> {
               : popToWizard(context);
         }
       });
-    } else if (widget.arguments.wizardActive || _isSpecialIssuanceSession(session)) {
+    } else if (widget.arguments.wizardActive || session.didIssueCredentialFromStore) {
       WidgetsBinding.instance?.addPostFrameCallback(
         (_) => widget.arguments.wizardActive ? popToWizard(context) : Navigator.of(context).pop(),
       );
```

Comment:
Is the `IssuanceSuccessScreen` not expected here if `session.didIssueCredentialFromStore`?

Reply:
To my knowledge this case only gets hit on mobile sessions so the IssuanceSuccessScreen should not be shown.

Reply:
Ah right, I see. That case is already covered on line 198. Maybe you can add a comment that explains this.